core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -3699,6 +3700,14 @@ void checkPeriodPredicate(Checker checker) {
         .ok(expected);
   }
 
+  @Test public void testInsertDialect() {
+    // See [CALCITE-3486] MysqlSqlDialect unparse ROW keyword does not exist
```

Comment:
You don't need this comment. If you actually want to see this test is testing `ROW`, you can rename this test as `testInsertWithRowDialect`.

Reply:
Thank you.
modified